packages/code-du-travail-frontend/src/outils/CommonIndemniteDepart/steps/Anciennete/store/validator/dateEntree.ts

```diff
@@ -15,7 +17,11 @@ export const getDateEntreeErrors = (
     return errors;
   }
   const dEntree = parse(state.dateEntree);
-  if (isBefore(dEntree, BEGINNING)) {
+  if (
+    !isValid(dEntree) ||
```

Comment:
je penses que ça sert rien le isValid, parce que notre date venant d'un inputDate, elle est forcément valid.
https://date-fns.org/v3.6.0/docs/isValid "Invalid Date is a Date, whose time value is NaN."